_zigbee/Sonoff_SNZB-04.md

```diff
@@ -13,6 +13,7 @@ link: https://www.itead.cc/sonoff-snzb-04-zigbee-wireless-door-window-sensor.htm
 link2: https://www.aliexpress.com/item/1005001275204366.html
 link3: https://www.banggood.com/SONOFF-SNZB-04-ZB-Wireless-Door-or-Window-Sensor-Enable-Smart-Linkage-Between-SONOFF-ZBBridge-and-WiFi-Devices-via-eWeLink-APP-p-1715994.html
 link4: https://www.domadoo.fr/fr/peripheriques/5320-sonoff-capteur-d-ouverture-de-portefenetre-zigbee.html
+link5: https://www.mylocalbytes.com/products/sonoff-snzb-04-zigbee-wireless-door-window-sensor
```

Comment:
there is no link5: available

Reply:
Oh no :(
I'll remove my addition for that file